EpikCord/__init__.py

```diff
@@ -148,7 +151,8 @@ async def fetch_guilds(
         if after:
             params["after"] = after
 
-        data = await (await self._http.get("/users/@me/guilds", params=params)).json()
+        guilds = await self._http.get("/users/@me/guilds", params=params)
+        data = await guilds.json()
```

Comment:
Reverse the names?

Reply:
That line was too long and the double await ain't good for typing, so i believe it's would be better to let a intermediate variable here

Reply:
No I mean the names, guilds isn't the requests data. so `data = await request`, `guilds = await data.json()`